public/style.css

```diff
@@ -211,7 +211,7 @@ button {
 }
 
 .skipped-correct { 
-  background-color: gray;
+  background-color: green;
```

Comment:
- When the user skips the question, lets make the background blue instead of green.